ref/general/packaging-deploying.adoc

```diff
@@ -14,79 +14,69 @@
+
+For more information about Open Liberty packaging options with Maven, see the link:https://github.com/OpenLiberty/ci.maven/blob/master/docs/package.md#package[`package` goal in the Maven documentation]. For more information about Open Liberty packaging options with Gradle, see the link:https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyPackage.md#libertypackage-task[`libertyPackage` task in the Gradle documentation].
+
+=== Packaging for containerization
+Use Maven or Gradle to package an application before you containerize it.
+Maven and Gradle both work well as part of continuous integration and continuous deployment (CI/CD) pipelines.
+
+When you package an application for containerization and cloud deployment, specify `usr` in the `include` parameter or attribute.
+This option packages only relevant files in the `wlp.user.dir` directory, effectively excluding the runtime binary files from the compressed file.
+
+=== Packaging into a WAR deployment artifact
+You can also choose to package your application into a WAR deployment artifact for deployment on an Open Liberty server.
+Use Maven or Gradle to package your application into a WAR file so that it can be manually deployed on Open Liberty.
+
+Specify `war` as the packaging type in your `pom.xml` file (for Maven) or in your `build.gradle` file (for Gradle). Then, package your application with Maven or Gradle. The result is a WAR deployment artifact that you can deploy on Open Liberty.
+
+=== Packaging into a runnable JAR file
+If you want to package locally, you can package your application and an Open Liberty installation into a link:/docs/ref/general/#runnablejarfiles.html[runnable JAR file].
 The JAR file is portable and ready to deploy both the application and server together.
-Packaging locally can be useful for debugging and testing during development.
-
-With Maven, run the following goal that overrides the `include` option to specify a minimum runnable JAR file as the packaging type:
-----
-mvn liberty:package -Dinclude=minify,runnable
-----
-
-With Gradle, include the following `packageLiberty` block in the `libertyPackage` task to specify a minimum runnable JAR file as the packaging type:
-----
-packageLiberty {
-    packageName = "MyPackage"
-    include = "minify,runnable"
-}
-----
-
-Each example overrides the `include` property to package your application and runtime into a runnable JAR file.
-The `minify` option specifies that the runnable JAR file contains only what you need to run your application so the application is as lightweight as possible.
-
-== Containerize
-After you create a server package by packaging your application with the Open Liberty runtime, you can containerize it for deployment.
+This local packaging method can be useful for debugging and testing during development.
+
+To package into a runnable JAR file, specify `runnable` as the package type in the `include` parameter or attribute.
```

Comment:
This feels confused because the first sentence in the section says this is for creating runnable jars, but this sentence makes it sound like a runnable jar is a different option from what was being talked about in the first sentence.